powerVCF.psm1

```diff
@@ -2558,28 +2558,24 @@ Export-ModuleMember -Function Get-VCFvRLI
 Function Invoke-VCFCommand {
     <#
         .SYNOPSIS
-        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SOS)
+        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SoS)
     
         .DESCRIPTION
-        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and 
-        subsequently execute elevated SOS commands using the root account. This is why both vcf and root password are
-        mandatory parameters. If passwords are not passed as parameters it will prompt for them.
+        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and subsequently 
+        execute elevated SOS commands using the root account. Bboth vcf and root password are mandatory parameters.
```

Comment:
Typo "Bboth"

Reply:
fixed just now in commit "Invoke-VCFCommand typos fixed on synopsis/description"